packages/api/tests/endpoints/collections/list-collections.js

```diff
@@ -86,34 +115,23 @@ test('CUMULUS-912 GET without pathParameters and with an invalid access token re
 test.todo('CUMULUS-912 GET without pathParameters and with an unauthorized user returns an unauthorized response');
 
 test.serial('default returns list of collections from query', async (t) => {
-  const stub = sinon.stub(EsCollection.prototype, 'query').returns({ results: [t.context.testCollection] });
-  const spy = sinon.stub(EsCollection.prototype, 'addStatsToCollectionResults');
-
   const response = await request(app)
     .get('/collections')
     .set('Accept', 'application/json')
     .set('Authorization', `Bearer ${jwtAuthToken}`)
     .expect(200);
 
   const { results } = response.body;
-  t.is(results.length, 1);
-  t.is(results[0].name, t.context.testCollection.name);
-  t.true(spy.notCalled);
-  stub.restore();
-  spy.restore();
+  t.is(results.length, 10);
```

Comment:
In general, this test seems fine. However, I'm wondering if we need to do any of the permutations that are in the granules endpoint tests, like pagination and limit, and if the meta needs to be included. Maybe that will come with a later ticket, but it feels like we should have some of that for collections when we're done with it.

Reply:
Hmm. That seems fair, but granules has it in both places. :thinking_face:

Reply:
Ok. Since it is covered in the other tests, I'm ok to leave it for now, and we can revisit hopefully with Jenny when we are wanting to merge the feature branch.